migrate/v0_15/migrate.go

```diff
@@ -191,3 +192,8 @@ func Committee(genesisState v0_14committee.GenesisState) v0_15committee.GenesisS
 	return v0_15committee.NewGenesisState(
 		genesisState.NextProposalID, committees, proposals, votes)
 }
+
+// Swap migrates new swap module state to a v0.15 swap genesis state
+func Swap(genesisState v0_15swap.GenesisState) v0_15swap.GenesisState {
+	return v0_15swap.NewGenesisState(genesisState.Params)
```

Comment:
Should we add the swap module account here as well?

Reply:
swap module account should be created in/checked to exist in `InitGenesis`. After that, the account state will be preserved in the `Auth` genesis state, so no need to do anything in Swap migration.

Reply:
Sounds good, I like that approach.  Is the reason we created the module account for hard in migration because we needed `minter` permissions?